### PR TITLE
feat(docs): Update falco.org homepage with new front end look/feel, logos, and CNCF slack

### PR DESCRIPTION
**Pull request by kris-nova**
Signed-off-by: Kris Nova <kris@nivenly.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

 /kind design

> /kind user-interface

/kind content

> /kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area blog

> /area documentation

**What this PR does / why we need it**:

Updating our logos and adopters on the home page to match adopters.md in the Falco repository. 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

This will render something like:

Screenshot from 2020-05-18 16-30-53

### CODE REVIEW

**Comment by kris-nova** [comment]
/unhold

This is ready to go!

**Comment by kris-nova** [comment]
closes https://github.com/falcosecurity/falco/issues/983

File: config.yaml [hunk 1]
```diff
@@ -33,58 +30,75 @@ languages:
...
+          buttonText4:  Examples
+          buttonLink4: https://falco.org/docs/examples/
       signup:
-        title: Join the mailing list
-        subtitle: Get involved with the community
+        title: Join the community
+        subtitle: Sign up for the development mailing list
         buttonText: Submit
+        slackText: Falco Slack Channels
+        slackLink: https://slack.k8s.io/
```
Comment (leogr) [inline config.yaml:93]: We will have to edit this and put the link to the channel

**Comment by Dotan1** [comment]
@kris-nova thanks for adding Logz.io to the vendors list. is that also going to be reflected on the ADOPTERS.md?

**Comment by leodido** [comment]
@Dotan1 feel free to send a PR to the main repo with addition to ADOPTERS.md file

**Pull request merged**